submitted_models/marble_hd2_sensor_config_1/model.sdf

```diff
@@ -154,20 +154,57 @@
           <specular>1.0 1.0 1.0</specular>
           <pbr>
             <metal>
-              <albedo_map>model://MARBLE_HD2_SENSOR_CONFIG_1/materials/textures/HD2_Albedo.png</albedo_map>
-              <metalness_map>model://MARBLE_HD2_SENSOR_CONFIG_1/materials/textures/HD2_Metalness.png</metalness_map>
-              <roughness_map>model://MARBLE_HD2_SENSOR_CONFIG_1/materials/textures/HD2_Roughness.png</roughness_map>
+              <albedo_map>materials/textures/HD2_Albedo.png</albedo_map>
+              <metalness_map>materials/textures/HD2_Metalness.png</metalness_map>
+              <roughness_map>materials/textures/HD2_Roughness.png</roughness_map>
             </metal>
           </pbr>
           <!-- fallback to script if no PBR support-->
           <script>
-            <uri>model://MARBLE_HD2_SENSOR_CONFIG_1/materials/scripts/</uri>
-            <uri>model://MARBLE_HD2_SENSOR_CONFIG_1/materials/textures/</uri>
+            <uri>materials/scripts/</uri>
+            <uri>materials/textures/</uri>
             <name>HD/HD2_Diffuse</name>
           </script>
         </material>
       </visual>
 
+      <!-- thermal camera -->
+      <visual name="thermal_camera_visual">
+        <pose>0.435 -.01 0.125 0 0 0</pose>
+        <geometry>
+          <box>
+            <size>0.02 0.025 0.025</size>
+          </box>
+        </geometry>
+      </visual>
+      <sensor name="thermal_camera" type="thermal">
+        <pose>0.435 -.01 0.125 0 0 0</pose>
+        <camera name="thermal_camera">
+          <horizontal_fov>1.047</horizontal_fov>
+          <lens>
+            <intrinsics>
+              <!-- fx = fy = width / ( 2 * tan (hfov / 2 ) ) -->
+              <fx>332.62963</fx>
+              <fy>332.62963</fy>
+              <!-- cx = ( width + 1 ) / 2 -->
+              <cx>192.5</cx>
+              <!-- cy = ( height + 1 ) / 2 -->
+              <cy>144.5</cy>
+              <s>0</s>
+            </intrinsics>
+          </lens>
+          <image>
+            <width>384</width>
+            <height>288</height>
+          </image>
+          <clip>
+            <near>0.1</near>
+            <far>100</far>
+          </clip>
```

Comment:
I couldn't find any clipping information in the camera specs, so I am going with these values for now. Let me know if they should be updated to something different, or if anyone finds this information in the camera specs somewhere.